freqtrade/data/history/history_utils.py

```diff
@@ -116,6 +120,28 @@ def load_data(datadir: Path,
     return result
 
 
+def optimize_dtypes(data: DataFrame) -> None:
```

Comment:
While taking a different approach - i don't really see this differ too much from `reduce_dataframe_footprint` (in `converter.py`)? 

Maybe that function can be improved instead of introducing a new function doing something VERY similar somewhere unfitting (this function shouldn't be in history_utils for sure).

Reply:
Haven't seen that function before. yes, the idea is the same. improving that function would work too.

Reply:
> Haven't seen that function before.

imageined :) 
that's why we do reviews ;)